### ISSUE TITLE
[issue]: manjaro secure boot issue 

### ISSUE THREAD

**Issue by bedman80** [ventoy/Ventoy#1078]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.51

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### BIOS Mode

UEFI Mode

### Partition Style

MBR

### Disk Capacity

64GB

### Image file checksum (if applicable)

Yes.

### Image file download link (if applicable)

https://download.manjaro.org/xfce/21.1.0/manjaro-xfce-21.1.0-210817-linux513.iso

### What happened?

Tried https://download.manjaro.org/xfce/21.1.0/manjaro-xfce-21.1.0-210817-linux513.iso

It does not boot von Lenovo Ideapad5 with secure boot enabled 

![IMG_20210828_165537](https://user-images.githubusercontent.com/89697799/131222913-45e18857-5161-4668-ab19-c9fb9756fcc2.jpg)

**Comment by ventoy** [comment]
Can it boot OK if you use Rufus to burn the same ISO to the same USB and boot with secure boot enabled?

**Comment by TuoBaed** [comment]
yes, still a problem, but I am not sure it is ventoy's problem or manjaro's problem. But Fedora 36 is installed correctly.
